docs/experimental-features.md

```diff
@@ -365,3 +365,30 @@ ipfs config --json Experimental.ShardingEnabled true
 
 - [ ] Make sure that objects that don't have to be sharded aren't
 - [ ] Generalize sharding and define a new layer between IPLD and IPFS
+
+---
+
+## IPNS pubsub
+
+### In Version
+
+0.4.13
```

Comment:
Was it in this version? I thought this only made it into 0.4.14.

Reply:
uhm ok, not sure either.